Validators/user.py

```diff
@@ -12,6 +12,7 @@ class User(BaseModel):
 class UserLogin(BaseModel):
     mail: str
     password: str
+    warderobe: Optional[str]
```

Comment:
Jak w poprzednim komentarzu, jeśli zmienimy wejście to nie będzie też trzeba modyfikować Walidatora